src/controllers/SubscribeController.js

```diff
@@ -0,0 +1,17 @@
+import LoggingService from '../services/LoggingService';
+import GenericResponse from '../utilities/GenericResponse';
+import SubscribeService from '../services/SubscribeService';
+
+const logger = LoggingService.getLogger('Subscribe');
+
+const subscribe = async (req, res) => {
+  logger.debug(`Subscribe called with ${JSON.stringify(req.body)}`);
+
+  logger.debug('Request Authorized Successfully.');
```

Comment:
do you need this logger

Reply:
Line8: is to verify the request body, can be removed
line10: is not needed, removing that